RFS/src/main/java/org/opensearch/migrations/bulkload/lucene/version_9/IgnoreElasticsearch816Codec.java

```diff
@@ -0,0 +1,77 @@
+package org.opensearch.migrations.bulkload.lucene.version_9;
+
+import shadow.lucene9.org.apache.lucene.codecs.*;
+
+/**
+ * Codec fallback for Elasticsearch 8.16+ segment formats.
+ *
+ * <p>This class provides a dummy implementation for "Elasticsearch816" to avoid runtime
+ * errors when Lucene 9 attempts to load this postings format from snapshot-based
+ * segment metadata during document migration.</p>
+ *
+ * <p>Registered via Lucene's SPI to allow dynamic loading based on Codec name
+ * stored in segment metadata.</p>
+ *
+ */
+public class IgnoreElasticsearch816Codec extends Codec {
```

Comment:
Based on the comment lets rename this to Elasticsearch816CodecFallback, does that seem like a better name?